### PR TITLE
Added OOP class

### PR DESCRIPTION
**Pull request by RZEROSTERN**
I've added a OOP version of the main script. It works this way:

1. Create a new EmailVerifier instance assignating as params the email
to test and the email that will be used as source. Ex:

$verifier = new EmailVerifier('test@domain.com', 'source@domain.com');

2. Use validate() for get the result of the testing. It'll be returned
as an array with the status and the details of the testing.

I hope this small class could be part of the branch as an OOP development alternative.
Greetings !!!

### CODE REVIEW

File: EmailVerifier.class.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+		 *	@return Result of the test into an array.
+		 */
+		private function connect(){
+			$connect = @fsockopen($this->mx_ip, 25);
+
+			if($connect){ 
+				if(preg_match("/^220/i", $out = fgets($connect, 1024))){
+					fputs ($connect , "HELO $mx_ip\r\n"); 
+					$out = fgets ($connect, 1024);
+					$details .= $out."\n";
```
Comment (abdullahnaseer) [inline EmailVerifier.class.php]: var $details not exist.
remove the dot before equal sign or first declare this at start

File: EmailVerifier.class.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+		 *	connect
+		 *	The connector. It connects to SMTP for test the given email.
+		 *	@return Result of the test into an array.
+		 */
+		private function connect(){
+			$connect = @fsockopen($this->mx_ip, 25);
+
+			if($connect){ 
+				if(preg_match("/^220/i", $out = fgets($connect, 1024))){
+					fputs ($connect , "HELO $mx_ip\r\n");
```
Comment (abdullahnaseer) [inline EmailVerifier.class.php]: mx_ip is working without error but $mx_ip not working proper.

Comment (RZEROSTERN) [inline EmailVerifier.class.php]: The mx_ip works as a class variable, so you must use it as $this->mx_ip

Comment (abdullahnaseer) [inline EmailVerifier.class.php]: Thankyou.

Comment (beckybeq) [inline EmailVerifier.class.php]: nice (y)

**Comment by hbattat** [comment]
The code has been re-written in OO style with more flexibility and scalability. See the latest code. Thanks

**Pull request closed without merging**